search-sentiments-map/app.js

```diff
@@ -31,10 +31,14 @@ const trends = require('./routes/trends.js');
 // Use the trends router so that it can be fetched from the client-side scripts.
 app.use('/trends', trends.router);
 
-//trends.updateTrendsFunction();  // Uncomment this to get trends if none is in the datastore.
+// Uncomment the following line to get trends if none is in the datastore.
```

Comment:
Should be "if none _are_ in the datastore"

Reply:
Also capitalize Datastore, since you did that in the trends.js file.

Reply:
ok